astroid/__pkginfo__.py

```diff
@@ -15,7 +21,12 @@
 
 extras_require = {}
 install_requires = ['lazy_object_proxy', 'six', 'wrapt']
-extras_require[':python_version<"3.4"'] = ['enum34', 'singledispatch']
+
+if py_version < (3, 4) and setuptools_version < '21.0.0':
```

Comment:
@jayvdb Thanks for clarify

I added (new commit) the following comment:

> The first known release to support environment marker with range operators
    it is 17.1, see: https://setuptools.readthedocs.io/en/latest/history.html#id113

It is important the **range operators** because we are using a range with `python_version<"3.4"`